dbms/src/Core/Settings.h

```diff
@@ -104,12 +104,12 @@ struct Settings : public SettingsCollection<Settings>
     M(SettingBool, optimize_skip_unused_shards, false, "Assumes that data is distributed by sharding_key. Optimization to skip unused shards if SELECT query filters by sharding_key.") \
```

Comment:
Don't understant the changes of this file.

Reply:
I have made `_bytes` settings almost equal (slightly greater) to the analogous `_rows` settings in  marks count. Previously `_bytes` settings were chosen without any logic and performance tests had showed decrease in performance https://clickhouse-test-reports.s3.yandex.net/5628/3f7892ef50252bd473cdc627aade065f34767b46/performance_test/comparison_min_time_gcc_8.html. `merge_tree_max_rows_to_use_cache` didn't changed, just rewrtite mutliplication in default marks size.